skimage/feature/util.py

```diff
@@ -76,9 +77,10 @@ def plot_matched_features(
         The Axes object where the images and their matched features are drawn.
     keypoints_color : matplotlib color, optional
         Color for keypoint locations.
-    matches_color : matplotlib color, optional
-        Color for lines which connect keypoint matches. By default the
-        color is chosen randomly.
+    matches_color : matplotlib color or sequence thereof, optional
```

Comment:
What is a `matplotlib color`? Is this a string, a specific object, etc.? Can we point to the matplotlib docs?

Reply:
I added a reference to the supported color formats.